src/engraving/rendering/dev/slurtielayout.cpp

```diff
@@ -1656,7 +1656,11 @@ double SlurTieLayout::noteOpticalCenterForTie(const Note* note, bool up)
         double singleSlashWidth = note->symBbox(SymId::noteheadSlashHorizontalEnds).width();
         double noteWidth = note->width();
         double center = 0.20 * note->spatium() + 0.5 * (noteWidth - singleSlashWidth);
-        return up ? note->shape().right() - center : note->shape().left() + center;
+        Shape noteShape =  note->shape();
+        noteShape.remove_if([&](ShapeElement& s) {
+            return s.item()->pos().x() + s.item()->width() > note->pos().x() + note->width();
```

Comment:
I'm not 100% sure if the calculation is correct. As far as I know, the top left point of the bounding box is specified relatively to `pos()`.
So it seems correct when the top left point of the bounding box is `(0,0)`, but otherwise not. 
Maybe `s.item()->pos().x() + s.item()->bbox().right()` (and same for `note`) does the job? Please correct me if I'm wrong.